sound/soc/sof/ipc4-topology.c

```diff
@@ -2869,7 +2869,7 @@ static void sof_ipc4_put_queue_id(struct snd_sof_widget *swidget, int queue_id,
 static int sof_ipc4_set_copier_sink_format(struct snd_sof_dev *sdev,
 					   struct snd_sof_widget *src_widget,
 					   struct snd_sof_widget *sink_widget,
-					   int sink_id)
+					   struct snd_sof_route *sroute)
```

Comment:
@ujfalusi I guess this worked before without issues only because we never had a process type widget connected to a module copier with multiple output pins before right? Maybe its worth mentioning it

Reply:
@ranj063, likely that is the case, in nocodec topology we have gains after the module-copier branch and likely this was tested in environment where the process module happened to have base config extension.